scripts/floods/run.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
```

Comment:
I like shell scripts, but a 300+ line shell scripts feels a bit much to maintain without tests.  Especially if this is going to be the common infra for extraction from EE. So I wonder how much of this can be python perhaps? For instance, does the job create/waiting etc happen using python APIs?